Evergreen/Apps/Get-Ghostscript.ps1

```diff
@@ -0,0 +1,33 @@
+Function Get-Ghostscript {
```

Comment:
Update with vendor name "Artifex", i.e., "ArtifexGhostscript"

Reply:
Updated to match Vendor.ApplicationName